beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/aws/S3BytesDeletedCalculatorTest.java

```diff
@@ -38,7 +38,7 @@ class S3BytesDeletedCalculatorTest {
 
   private long contentBytes = "content".getBytes().length;
   private String bucket = "bucket";
-  private String key1 = "db/table/id/partition1/file1";
+  private String key1 = "apiary-112945290135-us-east-1-user/mass/stream/output/segment_id=628b64be-188a-4d23-8e93-0dee521e8321/run_dt=2022-06-18";
```

Comment:
remove the account numbers and real paths from here as this is open source.

Reply:
I forgot about this, it was a test but i missed returning it back.

Reply:
@RickArt this information needs to disappear ASAP.